Code/GraphMol/RGroupDecomposition/RGroupUtils.cpp

```diff
@@ -87,13 +86,23 @@ bool setLabel(Atom *atom, int label, std::set<int> &labels, int &maxLabel,
     }
 
     atom->setProp<int>(RLABEL, label);
+    atom->setProp<int>(RLABEL_TYPE, static_cast<int>(type));
     labels.insert(label);
     maxLabel = (std::max)(maxLabel, label + 1);
     return true;
   }
   return false;
 }
 
+bool isAnyAtomWithMultipleNeighborsOrNotUserRLabel(const Atom &atom) {
+  if (atom.getAtomicNum()) return false;
+  if (atom.getDegree() > 1) return true;
```

Comment:
The comment in the header reads:
```
Returns true if the core atom is a dummy atom with a single connection and a user defined rgroup label
```
But the function seems to return `true`:
- if it is a dummy atom with multiple connections
- if it is a dummy atom with a single connection and no user-defined R-label

Reply:
You're correct.  I rewrote that function a few times and forgot to update the header.  I've put in a correct comment.